File: ToolKit/src/main/java/com/uday/android/util/DrawerListData.java
package com.uday.android.util;

public class DrawerListData
{
	public int icRes;
	public String title;
	public String header;
	
	public DrawerListData(String title,int icRes){
		this.icRes=icRes;
		this.title=title;
	}
	public DrawerListData(String header){
		this.header=header;
	}
}

File: app/src/main/java/com/uday/android/util/DrawerListData.kt
package com.uday.android.util

class DrawerListData {
    var icRes: Int = 0
    var title: String? = null
    var header: String? = null

    constructor(title: String, icRes: Int) {
        this.icRes = icRes
        this.title = title
    }

    constructor(header: String) {
        this.header = header
    }
}